package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jburton/markdown-spellcheck",
+  "name": "markdown-relative-file-spellcheck",
```

Comment:
Hi - just spotted this commit via a notification from the PR you opened.

Just so that you are aware, you do not have to install your fork to npm for us to be able to use it from the blog repo. The following will resolve just fine:

```
'mdspell' : 'jacoburton104/node-markdown-spellcheck' 
```

Reply:
although you'll need to reference the branch ...

Reply:
Thanks Colin! Silly mistake but at least I published my first npm package! lol hopefully I can remove the npm package now and do it the way you suggested, it seems a lot cleaner and easier!